jpeg/compress.go

```diff
@@ -175,6 +188,24 @@ func encodeYCbCr(cinfo *C.struct_jpeg_compress_struct, src *image.YCbCr, p *Enco
 	return
 }
 
+// encode image.RGBA
+func encodeRGBA(cinfo *C.struct_jpeg_compress_struct, src *image.RGBA, p *EncoderOptions) (err error) {
+	// Set up compression parameters
+	cinfo.image_width = C.JDIMENSION(src.Bounds().Dx())
+	cinfo.image_height = C.JDIMENSION(src.Bounds().Dy())
+	cinfo.input_components = 4
+	cinfo.in_color_space = C.JCS_EXT_RGBA
```

Comment:
It needs to check JCS_EXT_RGBA support.
(see: https://github.com/pixiv/go-libjpeg/blob/master/jpeg/decompress.go#L299)